dev/SapphireTest.php

```diff
@@ -30,12 +28,7 @@ class SapphireTest extends PHPUnit_Framework_TestCase {
 	 * @var FixtureFactory
 	 */
 	protected $fixtureFactory;
-	
-	/**
-	 * @var bool Set whether to include this test in the TestRunner or to skip this.
-	 */
-	protected $skipTest = false;
```

Comment:
When `markTestSkipped()` is placed in `setUp()`, it'll skip every test (https://phpunit.de/manual/current/en/incomplete-and-skipped-tests.html#incomplete-and-skipped-tests.skipping-tests). Which I've made sure is the case for the few core uses. You're correct in saying that if somebody discovered this flag, and used without also using `markTestSkipped()`, this test would now be executed. I think that's an acceptable change for a major version, particularly since its duplicating existing PHPUnit functionality. I've made this change more explicit in the upgrading docs though.